test/Benchmarks/OrleansBenchmarks/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using BenchmarkDotNet.Running;
+using OrleansBenchmarks.MapReduce;
+
+namespace OrleansBenchmarks
+{
+    class Program
+    {
+        private static readonly Dictionary<string, Action> _benchmarks = new Dictionary<string, Action>
+        {
+            ["MapReduce"] = () =>
+            {
+                Console.WriteLine("Running MapReduce benchmark");
+                var mapReduceBenchmark = new MapReduceBenchmark();
+                mapReduceBenchmark.BenchmarkSetup();
+                var stopWatch = Stopwatch.StartNew();
+                mapReduceBenchmark.Bench().Wait();
+                Console.WriteLine($"Elapsed milliseconds: {stopWatch.ElapsedMilliseconds}");
+                Console.ReadLine();
```

Comment:
Later maintainers will likely follow whatever pattern initially put in place.   More maintainable initial pattern will require less refactoring later.  

Suggest action logic be moved into a function for each action.  That will keep this initialization from growing into spaghetti hell over time.